test/irmin-pack/test_gc.ml

```diff
@@ -619,7 +619,6 @@ module Gc = struct
         "test"
         [
           ("mapping", 72);
-          ("suffix", 531);
```

Comment:
maybe update this field instead of removing it?

Reply:
It doesn't exist any longer after removing the stats line for "suffix" in the gc worker. I'm not sure if there should be a new stats for the suffix coming out of GC, but here is the context for what made this change https://github.com/mirage/irmin/pull/2126#discussion_r1009284289